### PR TITLE
Links to static content - French & Italian

### PR DESCRIPTION
**Pull request by thehabes**
Up for testing at http://paleo.rerum.io/TPEN-NL/
usr: bhaberbe@slu
pwd: 3

Open the pull down of tools and select "partial transcription" or "essay" to test the direct link to the correct static content.

### CODE REVIEW

File: web/js/newberry.js [hunk 1]
```diff
@@ -3085,7 +3085,16 @@ function splitPage(event, tool) {
             $("#compareSplit").css("width", splitWidthAdjustment);
             break
         case "partialTrans":
-            var currentCanvasLabel = transcriptionFolios[currentFolio - 1]["label"];
+            //var currentCanvasLabel = transcriptionFolios[currentFolio - 1]["label"];
+            var currentCanvasID = transcriptionFolios[currentFolio - 1]["@id"];
+            var utlID = "";
+            //We need to get the UTL canvas id for this particular canvas to support direct linking to the transcription for this object
+            utlID = currentCanvasID.substring(currentCanvasID.lastIndexOf("paleography:"), currentCanvasID.length - 1).replace("paleography:", "");
```
Comment (cubap) [inline web/js/newberry.js:3092]: I thought we wanted the first Index, it is just the case that for now we only have one match. Also, you may want a regex pattern for the match so we get only what we are expecting if there is a weird change.

At any rate, I think the `.replace` is excessive and you may just want `lastIndexOf()+12` instead

File: web/js/newberry.js [hunk 2]
```diff
@@ -3097,23 +3106,31 @@ function splitPage(event, tool) {
...
+            iframeDirectLink = buildIframeDirectLink("essay/" + utlID);
+            $("#partialTransSplit").children("iframe").attr("data_src", iframeDirectLink);
+            
+            /**
             if (currentCanvasLabel.split("_").length - 1 === 2) { //Must be in format like FP_000_000
                 //We need to get the UTL canvasID for this particular canvas to support direct linking to the essay for this object
                 utlID = currentCanvasLabel.substring(0, currentCanvasLabel.lastIndexOf("_")).toLowerCase().replace("ip_","").replace("fp_","");
-                iframeDirectLink = buildIframeDirectLink("essay/" + utlID + "?response_type=embed");
+                iframeDirectLink = buildIframeDirectLink("essay/" + utlID + ".md");
```
Comment (cubap) [inline web/js/newberry.js:3126]: nope. you made this change already but haven't pushed it up.

**Review by cubap — CHANGES_REQUESTED** [review]
I think there is just a missing commit so far, but most of this is good.

File: web/italianTranscription.html [hunk 3]
```diff
@@ -380,12 +380,12 @@ <h4 id="editBtns" title="Edit available XML Tags and Special Characters" class="
             
             <div id="startSplit" class="split">
                 <div style="text-align: center;" class="fullScreenTrans"><i class="fa fa-share fa-flip-vertical"></i></div>
-                <iframe class="loadingIframe" data_src="new-paleography?response_type=embed"></iframe>
+                <iframe class="loadingIframe" data_src="TODO?response_type=embed"></iframe>
```
Comment (thehabes) [inline web/italianTranscription.html:383]: Ah shoot I forgot I had been tempted to do both of these at the  same time, and so real early on I actually made some changes on this page.  They are correct at least...do you want me to undo these or keep them?

**Review by cubap — APPROVED** [review]

**Pull request merged**